### PR TITLE
Allow updating pids limit

### PR DESCRIPTION
**Pull request by mlaventure**
Signed-off-by: Kenfe-Mickael Laventure <mickael.laventure@gmail.com>

### CODE REVIEW

**Comment by dqminh** [comment]
LGTM

File: update.go [hunk 1]
```diff
@@ -228,6 +235,13 @@ other options are ignored.
 					*pair.dest = uint64(v)
 				}
 			}
+			if val := context.String("pids-limit"); val != "" {
```
Comment (crosbymichael) [inline update.go]: `context.Int()`?

File: update.go [hunk 2]
```diff
@@ -108,6 +108,10 @@ other options are ignored.
 			Name:  "memory-swap",
 			Usage: "Total memory usage (memory + swap); set '-1' to enable unlimited swap",
 		},
+		cli.StringFlag{
```
Comment (crosbymichael) [inline update.go]: `cli.IntFlag`

Comment (mlaventure) [inline update.go]: I want `int64` though, it doesn't exist unfortunately hence why the others are using `StringFlag`

**Comment by crosbymichael** [comment]
@mlaventure update test is broken

**Comment by mlaventure** [comment]
ping @crosbymichael  :)

**Comment by cyphar** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**